### PR TITLE
Adding the card effect for will.

### PR DESCRIPTION
**Pull request by petarpetkov**
This is will:

http://thronesdb.com/card/02001

### CODE REVIEW

File: server/game/cards/characters/02/will.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                    onSelect: (p, card) => this.onCardSelected(p, card)
+                });
+            }
+        });
+    }
+
+    onCardSelected(player, card) {
+        player.sacrificeCard(card);
+
+        this.game.addMessage('{0} uses {1} to sacrifice {2}', player, this, card);
```
Comment (DukeTax) [inline server/game/cards/characters/02/will.js]: Messages for forced reactions are usually written in the style of 'Player is forced by card to...'.

**Comment by ystros** [comment]
Other than the message, looks good. 👍

**Comment by petarpetkov** [comment]
fixed

**Comment by cryogen** [comment]
Thanks!

**Pull request merged**